src/functions-templates/js/fauna-crud/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "fauna-crud",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

I don't believe bumping this version is required, as this is used for template purposes.

```suggestion
  "version": "1.0.0",
```